langlib/lang.__internal/src/main/ballerina/src/lang.__internal/stream-utils.bal

```diff
@@ -100,13 +100,23 @@ public isolated function getReturnType(any func) returns typedesc<Type> = @java:
 #
 # + strm - The stream
 # + return - An abstract object which is iterable
-public function getIteratorObj(stream<Type, ErrorType> strm) returns
+public isolated function getIteratorObj(stream<Type, ErrorType> strm) returns
     object {
-        public function next() returns record {|Type value;|}|ErrorType?;} |
+        public isolated function next() returns record {|Type value;|}|ErrorType?;} |
     object {
-        public function next() returns record {|Type value;|}|ErrorType?;
-        public function close() returns ErrorType?;
+        public isolated function next() returns record {|Type value;|}|ErrorType?;
+        public isolated function close() returns ErrorType?;
     } = @java:Method {
         'class: "org.ballerinalang.langlib.internal.GetIteratorObj",
         name: "getIteratorObj"
     } external;
+
+# Invoke a non-isolated function as an external function (workaround for isolated invocations).
+#
+# + func - function to invoke
+# + args - args for the function to invoke
+# + return - result of the invocation
+public isolated function invokeAsExternal(any func, any|error... args) returns any|error|() = @java:Method {
```

Comment:
Shouldn't this be a non-public function in the stream langlib instead? We don't need it elsewhere, right?

Shall we also change the return type to `any|error`, since `()` is already covered in `any`?

Reply:
> Shouldn't this be a non-public function in the stream langlib instead? We don't need it elsewhere, right?
- needed in both stream and query libs.

> Shall we also change the return type to any|error, since () is already covered in any?
- fixed in d97ca9cdac22409c90315440b107d70588065625